flexmeasures/ui/crud/assets/forms.py

```diff
@@ -170,8 +170,13 @@ def with_sensors(
         asset: GenericAsset,
         account_id: Optional[int],
     ) -> None:
-        self.with_price_senors(asset, account_id)
-        self.with_inflexible_sensors(asset, account_id)
+        if current_app.config.get("FLEXMEASURES_SHOW_FLEXCONTEXT_EDIT", True):
```

Comment:
As the default behavior is to show it, I would negate the name of the config, something like: `FLEXMEASURES_HIDE_FLEXCONTEXT_EDIT`.

Reply:
Agreed.